res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -173,29 +196,6 @@ limitations under the License.
         box-sizing: border-box;
     }
 
-    .mx_ThreadPanel_dropdown {
```

Comment:
These blocks for dropdown are only used in the header, so it can be moved inside it.